python/ray/workflow/common.py

```diff
@@ -244,6 +288,11 @@ def slugify(value: str, allow_unicode=False) -> str:
 
 
 class Workflow(Generic[T]):
+    """This class represents a workflow. It would either be a workflow that
```

Comment:
Please follow the pydoc style here: https://docs.ray.io/en/master/getting-involved.html#code-style